plugins/chat/assets/javascripts/discourse/services/chat-channels-manager.js

```diff
@@ -97,6 +97,9 @@ export default class ChatChannelsManager extends Service {
   }
 
   remove(model) {
+    if (!model) {
```

Comment:
Kind of unrelated but when leaving a channel from drawer it would show an error due to model being `null` so added a guard clause to prevent that